indico/modules/rb_new/client/js/common/bookings/BookingEventLink.jsx

```diff
@@ -0,0 +1,89 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Message, Placeholder, Segment, Icon} from 'semantic-ui-react';
+import getBookingEventData from 'indico-url:rooms_new.booking_event_data';
+import {Translate} from 'indico/react/i18n';
+import {indicoAxios} from 'indico/utils/axios';
+import camelizeKeys from 'indico/utils/camelize';
+
+import './BookingEventLink.module.scss';
+
+
+export default class BookingEventLink extends React.Component {
```

Comment:
I think this can be a `PureComponent`

Reply:
What's the advantage of that?

Reply:
the component only updates when state/props changed instead of (by default) whenever the parent updates

Reply:
Is that still the case? [The docs](https://reactjs.org/docs/react-api.html#reactpurecomponent) are not very clear about that.

Reply:
Anyway, changed.